app/models/event.rb

```diff
@@ -106,4 +107,10 @@ def as_json(*)
   def schedule_notifications
     NotificationService.event_schedule_notifications(self)
   end
+
+  def destroy_event_users
```

Comment:
We use soft delete for this, so the event isn't really destroyed. It is however a bit strange that the last (almost) query that rails runs on `event.destroy!` is
``` 
SQL (8.6ms)  UPDATE `events` SET `events`.`deleted_at` = '2017-06-29 11:59:45' WHERE `events`.`id` = 9
```
and the queries from `after_destroy` are executed before this! One would think that `after_destroy` runs after the event was destroyed, but that does't appear to be the case here...